### ISSUE TITLE
Missing "policyDefinitionReferenceId" in PolicySetDefinitions

### ISSUE THREAD

**Issue by wendlandreas** [Azure/alz-monitor#170]
The policy definitions in the initiatives (policy set definitions) are missing the field "policyDefinitionReferenceId".
The result is that in the Azure portal in some places randomly generated policyDefinitionReferenceIds are visible, for example on the "policies" tab of the initiative definition (but also in other places like when selection the policy in "create remediation task").

![image](https://github.com/Azure/alz-monitor/assets/16876435/a7171fc7-8938-489b-a886-4d9fb24781ad)

The fix should be easy, each policy in the policy set definition should contain a "policyDefinitionReferenceId" with a descriptive name.
``` json
{
  "policyDefinitionReferenceId": "ALZM_DeployActivitylogServiceHealthSecurityAdvisory",
  "policyDefinitionId": "[concat('/providers/Microsoft.Management/managementGroups/',managementGroup().name, '/providers/Microsoft.Authorization/policyDefinitions/Deploy_activitylog_ServiceHealth_SecurityAdvisory')]",
  "parameters": {
    "enabled": {
      "value": "[[parameters('svcHlthSecAdvisoryAlertState')]"
    },
    "alertResourceGroupName": {
      "value": "[[parameters('ALZMonitorResourceGroupName')]"
    },
    "alertResourceGroupTags": {
      "value": "[[parameters('ALZMonitorResourceGroupTags')]"
    },
    "alertResourceGroupLocation": {
      "value": "[[parameters('ALZMonitorResourceGroupLocation')]"
    }
  }            
}
```

**Comment by jfaurskov** [comment]
Hi @wendlandreas thank you for taking the time to contribute to this. This is certainly something that we need to fix. I created a user story in our internal backlog. Of course we would welcome any contributions you might want to make, so if you have the bandwidth and inclination feel free to create a pull request towards this.  :-)

**Comment by jfaurskov** [comment]
Awesome! Let's go with the same naming convention that is used in our other projects, i.e. the prefix to parameters, so the Service health security advisory should be svcHlthSecAdvisory (from svcHlthSecAdvisoryAlertState), service health Incident is SvcHlthIncident (from SvcHlthIncidentAlertState) and so on. Does that make sense?

**Comment by wendlandreas** [comment]
Sure sounds good.
I will create a pull request as soon as I'm done.

**Comment by jfaurskov** [comment]
resolved in #190
